### DISCUSSION TITLE
Wiggling spinner

### DISCUSSION THREAD

**Discussion by tonmtom9** [ppy/osu#22877] [Ideas] [answered]
I don't know if this is wanted behaviour or not, but when I try to "wiggle" with spinner. The game registers it as spinning with decent speed. 

I think there should be some checks in place to verify, you actually spinning the spinner.

**Comment by Axilotl17** [comment]
i think this is known/intentional. It's enough movement, and its more awkward to have the wiggle not spin at all. the movement while not exactly being spinning is movement enough to "spin" the spinner.

**Reply by tonmtom9** [reply]
Should be at least more limited? So it wouldn’t spin that much?

**Reply by Joehuu** [reply]
This may be a duplicate of https://github.com/ppy/osu/issues/8224, but for reference, can you show a video of the "wiggle" movement?

**Reply by tonmtom9** [reply]
Just uploaded it

**Comment by tonmtom9** [comment]
Video reference

https://user-images.githubusercontent.com/60811343/225574172-384cc391-7748-4535-b8ac-80f7ab7bbde1.mp4

**Reply by integerrr** [reply]
Yes this is known for a long while

**Reply by Axilotl17** [reply]
intended, i believe.

**Comment by Joehuu** [comment, marked as answer]
This is indeed a duplicate of https://github.com/ppy/osu/issues/8224.
